server/util/redact/redact_test.go

```diff
@@ -0,0 +1,165 @@
+package redact_test
+
+import (
+	"testing"
+
+	"github.com/buildbuddy-io/buildbuddy/proto/build_event_stream"
+	"github.com/buildbuddy-io/buildbuddy/proto/command_line"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/testenv"
+	"github.com/buildbuddy-io/buildbuddy/server/util/redact"
+	"github.com/stretchr/testify/assert"
+)
+
+func singleFile() *build_event_stream.File {
+	return &build_event_stream.File{
+		Name: "afile",
+		File: &build_event_stream.File_Uri{
+			Uri: "213wZJyTUyhXkj381312@uri",
+		},
+	}
+}
+
+func singleFiles() []*build_event_stream.File {
+	return []*build_event_stream.File{
+		singleFile(),
+	}
+}
+
+func TestRedact(t *testing.T) {
```

Comment:
nit: this test is on the long side, consider splitting it up into multiple test functions

Reply:
Done